src/components/Organisms/Donate/Form/useClickOutside.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react';
+
+// Create function to reset money buy after leaving input empty and clicking outside
+export default function useClickOutside(
+  amountRef,
+  setAmountDonate,
+  givingType,
+  singleGiving,
+  errorMsg,
+  regularGiving
+) {
+  useEffect(() => {
+    /**
+     * Alert if clicked on outside of element
+     */
+    function handleClickOutside(event) {
```

Comment:
Hmm could we move the function declaration out of the useEffect so we're not recreating  it unnecessarily each time?